src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx

```diff
@@ -81,7 +82,7 @@ export class ArtworkDetailsAboutTheWorkFromPartner extends React.Component<
                   }
                   meta={locationNames}
                   imageUrl={imageUrl}
-                  initials={partner.initials}
+                  initials={showImage && partner.initials}
```

Comment:
This is a little confusing to read ("if there's an image, show the initials") -- can we add a comment describing the behavior?

Reply:
Sure, it is confusing a bit because the default for the Avatar is if either initials or imageUrl is passed - display logo. I think it might be lass confusing if I keep those in a separate variable up top. Will update. One min.